bundles/org.openhab.binding.exec/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -10,6 +10,8 @@
 
 		<channels>
 			<channel id="output" typeId="output"/>
+			<channel id="stdout" typeId="stdout"/>
+			<channel id="stderr" typeId="stderr"/>
```

Comment:
Please add upgrade instructions for an automatic integration of these new channels for users using managed things.

Reply:
There's no need for any upgrade instructions. The old `output` channel behaves like it always did. These channels are added, exactly not to break anything, but they allow retrieving just the information you're interested in (many redirect `stderr` to null/nil to avoid having to deal with the in `output`, these can just use `stdout` instead).

Did I misunderstand, did you mean something else?

Reply:
Ok, just give me a hint how I can figure out what to do, and I'll do it.